cluster-autoscaler/cloudprovider/aws/README.md

```diff
@@ -131,6 +155,64 @@ spec:
             path: "/etc/ssl/certs/ca-certificates.crt"
 ```
 
+### Auto-Discovery Setup
+
+As of today, docker images including the support for `--node-group-auto-discovery` is not yet published to official repository.
+Please checkout the latest source of this project locally and run `REGISTRY=<your docker repo> make release` to build and push an image yourself.
+Then, a manifest like below would run a cluster-autoscaler which auto-discovers ASGs tagged with `k8s.io/cluster-autoscaler/enabled` to be node groups.
+Please notice that there're no `--nodes` flags passed to cluster-autoscaler anymore.
+ 
+```yaml
+---
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: cluster-autoscaler
+  namespace: kube-system
+  labels:
+    app: cluster-autoscaler
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: cluster-autoscaler
+  template:
+    metadata:
+      labels:
+        app: cluster-autoscaler
+    spec:
+      containers:
+        - image: <your docker repo>/cluster-autoscaler:dev
+          name: cluster-autoscaler
+          resources:
+            limits:
+              cpu: 100m
+              memory: 300Mi
+            requests:
+              cpu: 100m
+              memory: 300Mi
+          command:
+            - ./cluster-autoscaler
+            - --v=4
+            - --stderrthreshold=info
+            - --cloud-provider=aws
+            - --skip-nodes-with-local-storage=false
+            - --expander=least-waste
+            - --node-group-auto-discovery=asg:tag=k8s.io/cluster-autoscaler/enabled
+          env:
+            - name: AWS_REGION
+              value: us-east-1
+          volumeMounts:
```

Comment:
@andrewsykim Sounds great 👍 
@mwielgus @MaciekPytel Can we add the `ca-certificates` package into the image in an another PR?